rs3-full-boss-guides/magister.txt

```diff
@@ -1,178 +1,310 @@
-    • 5 Mixtures hitting your Blood Reaver <:reaverpouch:839903693837959228> will kill it, 6 Mixtures will kill your Ripper Demon <:ripperpouch:703581275453128714>, and 10 will kill your Nihil <:shadownihil:854475441946230855> <:bloodnihil:854475442169053194> <:smokenihil:854475442227642388>.
+__**Barge/Stand Still**__
+.img:https://i.imgur.com/ier7amn.gif
+⬥ Magister will do one of two attacks depending on where the player is standing.
+    • **Player in MD**: Magister says `Stand still!` and hits once, dealing heavy (~2.5k) melee damage, stunning the player and disabling protection prayers briefly.
+    • **Player not in MD**: Magister says `You dare fight me?`, *and after 1.8s* barges the player's location dealing heavy melee damage in a 5x5 AoE, and disabling protection prayers briefly.
+⬥ Afterwards, the Magister may sometimes dash away from the player, if he got too close to the obelisk.
+
+⬥ **How to deal with it:**
+    • **If in MD**: Use Freedom <:freedom:535541258240786434> and pray Magic <:DeflectMage:544195487926845462>
+    • **If not in MD**: Quickly move 2 tiles *north* to avoid the attack, *or* move 1 tile in any direction and use Resonance <:res:535541258844635148> / Disruption Shield <:disrupt:535614336207552523>
+
+.
+__**Minions**__
+.img:https://i.imgur.com/4rbODea.gif
+⬥ **Magister says `Those who fall shall rise again!` and summons two random soul devourer minions with 10k health each.**
+⬥ Minions remain until killed or the boss dies.
+⬥ **How to deal with it:**
+    • Ignore them if the fight is almost over.
+    • Kill them otherwise.
+
 .
-⬥ Pray at War's Altar in War's Retreat between trips to restore your familiar's hitpoints.
+__**Unstable Mixture**__
+.img:https://i.imgur.com/EqOep9I.gif
+⬥ **Magister says `Dodge this!` and throws a bomb near the player.**
+⬥ Bomb has a bar that fills up, then explodes in a **5x5 AoE dealing 4000 typeless damage** (2000 to familiars).
+⬥ **How to deal with it:**
+    • Click on the bomb to kick it away.
+    • Or just move away from the bomb.
 
 .
-> **__Strategy__**
-.tag:strats
-**__General__**
-⬥ Avoid using high damaging abilities like Tuska's Wrath <:Tuskas:513201065513058306>, Wild Magic <:wm:535533809978966037>, Snap Shot <:snap:535534127131394088> etc. while the boss is slightly above 150k, 100k, and 50k hp as he will absorb any extra damage dealt to reach those exact numbers.
-    • Damage is counted towards progressing phases before truncation.
+__**Asphyxiate**__
+.img:https://i.imgur.com/WTSVZKC.gif
+⬥ **Magister says `Your sacrifice allows me to live!` and asphyxiates <:asphyx:535533833072672778> the player, deaing 3 hits of 1500-3000.**
+    • This does not stun the player.
+    • Each hit heals him for the damage dealt.
+⬥ **How to deal with it:**
+    • Use a melee threshold ability to interrupt it.
+    • Or move out of his attack range.
+    • Or simply ignore it.
```

Comment:
No need to start with "Or", they're already different bullets.
Maybe rename the section to "Ways of dealing with it" instead, so it doesn't look like a step by step
Think this was done elsewhere too